exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -528,11 +528,13 @@ func (b *Bithumb) GetOrderInfo(ctx context.Context, orderID string, pair currenc
 			Pair:            pair,
 		}
 
-		if orders.Data[i].Type == "bid" {
+		switch orders.Data[i].Type {
```

Comment:
🗻 OOS: Thinking that `Type` could be a order.Type and then this wouldn't be a string comparison.
Did the same on websockets recently with @Beadko 's work.
Maybe she wants to snipe this in a separate PR

Reply:
Deferred since I've got a cleanup branch which has this in mind already